Misc/NEWS.d/next/Core and Builtins/2018-09-28-00-52-28.bpo-27195.0dwSQz.rst

```diff
@@ -0,0 +1,6 @@
+Access to the ``format`` of a ``memoryview`` object could crash if the
+corresponding field in the buffer structure was not set. Per documentation
+of the new-style buffer protocol, an unset ``format`` field is to be
+interpreted as ``"B"``, so that is now returned in this case.
+
```

Comment:
Please remove the empty line. Unfortunately multiparagraph news entries are not supported.